All Travis builds fail with error TS7033

## :bug: Bug Report

### What is the problem?
<!--
Describe in as much detail as possible the problem that you have identified.
Please identify clearly what example the problem relates to.
-->
All builds fail on a random typescript example (the first to build) with this error:
```ts
node_modules/@aws-cdk/core/lib/stack.d.ts:425:17 - error TS7033: Property 'assetParameters' implicitly has type 'any', because its get accessor lacks a return type annotation.

425     private get assetParameters();
                    ~~~~~~~~~~~~~~~
```

__RELATED TO:__ [cdk#5497](https://github.com/aws/aws-cdk/issues/5497)

### Reproduction Steps
<!--
Whenever possible, add reference to the minimal reproduction of the issue, as
this will greatly help with identifying the root cause of the problem, and
ensuring a candidate fix is working.

If your minimal reprodiction is large, you may provide references to a [gist][1]
or any other publicly viewable location.
-->

Attempt to build any current example in repo with latest CDK version.

### Verbose Log
<!--
If the output is very large, you may store it in a [gist][1]. In any case,
please review the contents of the log you are about to submit and edit any
confidential information out.

[1]: https://gist.github.com
-->

### Environment

  - **CDK CLI Version:** <!-- Output of `cdk version` -->
  - **Example:** <!-- Name of the example in question -->
  - **Example Version:** <!-- Version of the module in question -->
  - **OS:** <!-- [all | Windows 10 | OSX Mojave | Ubuntu | etc... ] -->
  - **Language:** <!-- [all | TypeScript | Java | Python ] etc... ] -->


### Other information 
<!-- e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. associated pull-request, stackoverflow, gitter, etc -->

--- a/typescript/api-cors-lambda-crud-dynamodb/package.json
+++ b/typescript/api-cors-lambda-crud-dynamodb/package.json
@@ -15,7 +15,7 @@
   "license": "MIT",
   "devDependencies": {
     "@types/node": "^10.17.0",
-    "typescript": "~3.6.4"
+    "typescript": "~3.7.2"
   },
   "dependencies": {
     "@aws-cdk/aws-apigateway": "*",

--- a/typescript/application-load-balancer/package.json
+++ b/typescript/application-load-balancer/package.json
@@ -16,7 +16,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/node": "^10.17.0",
-    "typescript": "~3.6.4"
+    "typescript": "~3.7.2"
   },
   "dependencies": {
     "@aws-cdk/aws-autoscaling": "*",

--- a/typescript/appsync-graphql-dynamodb/package.json
+++ b/typescript/appsync-graphql-dynamodb/package.json
@@ -15,7 +15,7 @@
   "license": "MIT",
   "devDependencies": {
     "@types/node": "^10.17.0",
-    "typescript": "~3.6.4"
+    "typescript": "~3.7.2"
   },
   "dependencies": {
     "@aws-cdk/aws-appsync": "*",

--- a/typescript/classic-load-balancer/package.json
+++ b/typescript/classic-load-balancer/package.json
@@ -16,7 +16,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/node": "^10.17.0",
-    "typescript": "~3.6.4"
+    "typescript": "~3.7.2"
   },
   "dependencies": {
     "@aws-cdk/aws-autoscaling": "*",

--- a/typescript/custom-resource/package.json
+++ b/typescript/custom-resource/package.json
@@ -16,7 +16,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/node": "^10.17.0",
-    "typescript": "~3.6.4"
+    "typescript": "~3.7.2"
   },
   "dependencies": {
     "@aws-cdk/aws-cloudformation": "*",

--- a/typescript/ecs/cluster/package.json
+++ b/typescript/ecs/cluster/package.json
@@ -16,7 +16,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/node": "^8.10.38",
-    "typescript": "^3.2.4"
+    "typescript": "~3.7.2"
   },
   "dependencies": {
     "@aws-cdk/aws-ec2": "*",

--- a/typescript/ecs/cross-stack-load-balancer/package.json
+++ b/typescript/ecs/cross-stack-load-balancer/package.json
@@ -16,7 +16,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/node": "^8.10.38",
-    "typescript": "^3.2.4"
+    "typescript": "~3.7.2"
   },
   "dependencies": {
     "@aws-cdk/aws-ec2": "*",

--- a/typescript/ecs/ecs-network-load-balanced-service/package.json
+++ b/typescript/ecs/ecs-network-load-balanced-service/package.json
@@ -16,7 +16,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/node": "^8.10.38",
-    "typescript": "^3.2.4"
+    "typescript": "~3.7.2"
   },
   "dependencies": {
     "@aws-cdk/aws-ec2": "*",

--- a/typescript/ecs/ecs-service-with-advanced-alb-config/package.json
+++ b/typescript/ecs/ecs-service-with-advanced-alb-config/package.json
@@ -16,7 +16,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/node": "^8.10.38",
-    "typescript": "^3.2.4"
+    "typescript": "~3.7.2"
   },
   "dependencies": {
     "@aws-cdk/aws-ec2": "*",

--- a/typescript/ecs/ecs-service-with-logging/package.json
+++ b/typescript/ecs/ecs-service-with-logging/package.json
@@ -16,7 +16,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/node": "^8.10.38",
-    "typescript": "^3.2.4"
+    "typescript": "~3.7.2"
   },
   "dependencies": {
     "@aws-cdk/aws-ec2": "*",

--- a/typescript/ecs/ecs-service-with-task-networking/package.json
+++ b/typescript/ecs/ecs-service-with-task-networking/package.json
@@ -16,7 +16,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/node": "^8.10.38",
-    "typescript": "^3.2.4"
+    "typescript": "~3.7.2"
   },
   "dependencies": {
     "@aws-cdk/aws-ec2": "*",

--- a/typescript/ecs/ecs-service-with-task-placement/package.json
+++ b/typescript/ecs/ecs-service-with-task-placement/package.json
@@ -16,7 +16,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/node": "^8.10.38",
-    "typescript": "^3.2.4"
+    "typescript": "~3.7.2"
   },
   "dependencies": {
     "@aws-cdk/aws-ec2": "*",

--- a/typescript/ecs/fargate-application-load-balanced-service/package.json
+++ b/typescript/ecs/fargate-application-load-balanced-service/package.json
@@ -16,7 +16,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/node": "^8.10.38",
-    "typescript": "^3.2.4"
+    "typescript": "~3.7.2"
   },
   "dependencies": {
     "@aws-cdk/aws-ec2": "*",

--- a/typescript/ecs/fargate-service-with-auto-scaling/package.json
+++ b/typescript/ecs/fargate-service-with-auto-scaling/package.json
@@ -16,7 +16,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/node": "^8.10.38",
-    "typescript": "^3.2.4"
+    "typescript": "~3.7.2"
   },
   "dependencies": {
     "@aws-cdk/aws-ec2": "*",

--- a/typescript/ecs/fargate-service-with-local-image/package.json
+++ b/typescript/ecs/fargate-service-with-local-image/package.json
@@ -16,7 +16,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/node": "^8.10.38",
-    "typescript": "^3.2.4"
+    "typescript": "~3.7.2"
   },
   "dependencies": {
     "@aws-cdk/aws-ec2": "*",

--- a/typescript/ecs/fargate-service-with-logging/package.json
+++ b/typescript/ecs/fargate-service-with-logging/package.json
@@ -16,7 +16,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/node": "^8.10.38",
-    "typescript": "^3.2.4"
+    "typescript": "~3.7.2"
   },
   "dependencies": {
     "@aws-cdk/aws-ec2": "*",

--- a/typescript/elasticbeanstalk/elasticbeanstalk-bg-pipeline/package.json
+++ b/typescript/elasticbeanstalk/elasticbeanstalk-bg-pipeline/package.json
@@ -16,7 +16,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/node": "^8.10.40",
-    "typescript": "^3.2.4",
+    "typescript": "~3.7.2",
     "aws-cdk": "*"
   },
   "dependencies": {

--- a/typescript/elasticbeanstalk/elasticbeanstalk-environment/package.json
+++ b/typescript/elasticbeanstalk/elasticbeanstalk-environment/package.json
@@ -16,7 +16,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/node": "^8.10.40",
-    "typescript": "^3.2.4",
+    "typescript": "~3.7.2",
     "aws-cdk": "*"
   },
   "dependencies": {

--- a/typescript/my-widget-service/package.json
+++ b/typescript/my-widget-service/package.json
@@ -16,7 +16,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/node": "^10.17.0",
-    "typescript": "~3.6.4"
+    "typescript": "~3.7.2"
   },
   "dependencies": {
     "@aws-cdk/aws-apigateway": "*",

--- a/typescript/resource-overrides/package.json
+++ b/typescript/resource-overrides/package.json
@@ -16,7 +16,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/node": "^10.17.0",
-    "typescript": "~3.6.4"
+    "typescript": "~3.7.2"
   },
   "dependencies": {
     "@aws-cdk/aws-autoscaling": "*",

--- a/typescript/static-site/package.json
+++ b/typescript/static-site/package.json
@@ -15,7 +15,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/node": "^10.17.0",
-    "typescript": "~3.6.4"
+    "typescript": "~3.7.2"
   },
   "dependencies": {
     "@aws-cdk/aws-certificatemanager": "*",

--- a/typescript/stepfunctions-job-poller/package.json
+++ b/typescript/stepfunctions-job-poller/package.json
@@ -16,7 +16,7 @@
   "license": "Apache-2.0",
   "devDependencies": {
     "@types/node": "^10.17.0",
-    "typescript": "~3.6.4"
+    "typescript": "~3.7.2"
   },
   "dependencies": {
     "@aws-cdk/aws-stepfunctions": "*",

